api/src/main/java/org/openmrs/module/pihcore/setup/InitializerSetup.java

```diff
@@ -51,6 +55,27 @@ public static void loadPreConceptDomains() {
         }
     }
 
+    /**
+     * The purpose of this method is to determine if there are any site-specific configuration files,
+     * and if so, to exclude them if they are not intended for the specific config in use.
+     * Any config files that contain a "-site-", and which do not end with the site name, are excluded
+     */
+    public static List<String> getExclusionsForLoader(Loader loader, Config config) {
+        List<String> exclusions = new ArrayList<>();
+        if (loader instanceof BaseFileLoader) {
+            BaseFileLoader ll = (BaseFileLoader) loader;
+            String site = config.getSite() == null ? "" : config.getSite().toLowerCase();
+            for (File f : ll.getDirUtil().getFiles("csv")) {
+                String filename = f.getName().toLowerCase();
+                if (filename.contains("-site-") && !filename.endsWith("-" + site + ".csv")) {
```

Comment:
@mseaton here's the change I made...

Reply:
Seems ok.  Why not require file to end with "-site-" + site + ".csv"?

Reply:
Makes sense, will change.